packages/arb-token-bridge-ui/src/util/fetchL2Gateways.ts

```diff
@@ -24,6 +25,7 @@ export async function fetchL2Gateways(l2Provider: Provider) {
   const l2DaiGateway = l2DaiGatewayAddresses[l2Network.chainID]
   const l2wstETHGateway = l2wstETHGatewayAddresses[l2Network.chainID]
   const l2LptGateway = l2LptGatewayAddresses[l2Network.chainID]
+  const novaMoonGateway = l2MoonGatewayAddresses[l2Network.chainID]
```

Comment:
I think we should keep it consistent with all the other stuff. Even though they need refactoring because it could be Orbit chains too, but then we can change all of them at once when we refactor.

```suggestion
  const l2MoonGateway = l2MoonGatewayAddresses[l2Network.chainID]
```